tests/unit/lms/tasks/email_digests_test.py

```diff
@@ -223,148 +224,52 @@ def send_instructor_email_digest(self, patch):
-                make_task_done(omit=["h_userid"]),
-                # A TaskDone with no created_before, this should be ignored.
-                make_task_done(omit=["created_before"]),
-                # A TaskDone with an invalid created_before, this should be ignored.
-                make_task_done(created_before="foo"),
-                # The TaskDone that we expect get_task_done() to return.
-                matching_task_done,
-            ]
-        )
-
+    def test_it(self, created_after, created_before, digest_service, h_userid):
         send_instructor_email_digest(
             h_userid=h_userid,
+            created_after=created_after.isoformat(),
             created_before=created_before.isoformat(),
             override_to_email=sentinel.override_to_email,
         )
 
         digest_service.send_instructor_email_digest.assert_called_once_with(
-            h_userid,
-            datetime.fromisoformat(matching_task_done.data["created_before"]),
-            created_before,
-            override_to_email=sentinel.override_to_email,
-        )
-
-    def test_it_when_theres_no_matching_TaskDone(
-        self, created_before, digest_service, h_userid
-    ):
-        send_instructor_email_digest(
-            h_userid=h_userid, created_before=created_before.isoformat()
-        )
-
-        digest_service.send_instructor_email_digest.assert_called_once_with(
-            h_userid, created_before - timedelta(days=7), created_before
-        )
-
-    def test_it_when_TaskDone_is_more_than_a_week_old(
-        self, created_before, db_session, digest_service, h_userid, make_task_done
-    ):
-        db_session.add(
-            make_task_done(
-                created_before=(created_before - timedelta(days=8)).isoformat()
-            )
-        )
-
-        send_instructor_email_digest(
-            h_userid=h_userid, created_before=created_before.isoformat()
-        )
-
-        digest_service.send_instructor_email_digest.assert_called_once_with(
-            h_userid, created_before - timedelta(days=7), created_before
-        )
-
-    def test_it_when_TaskDone_doesnt_have_tzinfo(
-        self, created_before, db_session, digest_service, h_userid, make_task_done
-    ):
-        created_before = created_before.replace(tzinfo=None)
-
-        db_session.add(
-            make_task_done(
-                created_before=(created_before - timedelta(days=8)).isoformat()
-            )
-        )
-
-        send_instructor_email_digest(
-            h_userid=h_userid, created_before=created_before.isoformat()
-        )
-
-        digest_service.send_instructor_email_digest.assert_called_once_with(
-            h_userid=h_userid,
-            # The task adds tzinfo if it was not already present in the DB
-            created_after=created_before.replace(tzinfo=timezone.utc)
-            - timedelta(days=7),
+            h_userid="test_h_userid",
+            created_after=created_after,
```

Comment:
Hard to see from the diff, but this is the same assertion as before but with kwargs and the `created_after` value is the same value that the test passed to the method, rather than a value from the `task_done` table